### ISSUE TITLE
Ribbon Replacement.

### ISSUE THREAD

**Issue by Offline-R503B** [WolvenKit/WolvenKit#459]
To keep information and the wiki centralized, I would like to replace the Ribbon with the Syncfusion one. 

A few pros :

Looks cleaner with the rest of the app, easy to use, many end user options from the get go.
A simplified mode that looks a lot like VS 
![image](https://user-images.githubusercontent.com/11785451/118341601-086eeb80-b520-11eb-855e-25856ef2b92c.png)
Working Context Menus

One issue (A visual studio one)
Basically :
1 : Startup app
2 : Open project
3 : minimize ribbon
4 : click ribbon item
App crashes (Only in debugging)
![image](https://user-images.githubusercontent.com/11785451/118312514-c4fd8880-b4f1-11eb-880e-c5839503f134.png)

I don't see this as a major blocker however as during dev I don't really see the need to 'minimize' the ribbon.

**Comment by Offline-R503B** [comment]
![image](https://user-images.githubusercontent.com/11785451/118319375-f464c300-b4fa-11eb-8d2f-dffe5b805f8c.png)
Preview

**Comment by Offline-R503B** [comment]
Side effect : Switching themes will require the app to restart. (Some Syncfusion elements do not refresh like the ribbon) 
This however would allow for a more proper theme selection and the removal of theme selection during first startup. (Moving theme creation to "Personalization" in settings for example)

**Comment by rfuzzo** [comment]
looks good - I can't see the difference :D

**Comment by Offline-R503B** [comment]
- [ ] Ribbon Buttons Porting
- [ ] Ribbon Commands Cleanup for use with #460.
- [ ] Hide Tools And Editors Groups based on GameType.
- [ ] Contextual Tabs for Tools and Editors.
- More to come as we need to investigate removing the shell.
- If possible we then we need to port the start page to new window.

**Comment by Offline-R503B** [comment]
All is ready to be merged to Dev

**Comment by Offline-R503B** [comment]
Added in #466  (some buttons do need to be sorted but that can be done later , non issue)
